.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-*       @kodadot/code-review-guild
+*			@kodadot/code-review-guild
+/libs/ui	@roiLeo
```

Comment:
```suggestion
/libs/ui	@kodadot/internal-dev 
```

Reply:
then this line is unnecessary, I wanted to be added as reviewer each code change in libs/ui